static/templates/left_sidebar.hbs

```diff
@@ -69,6 +55,31 @@
                 <span class="arrow drafts-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
             </li>
         </ul>
+
+    <div id="pm_stream_scroll_container" class="scrolling_list" data-simplebar>
```

Comment:
Maybe `left_sidebar_scroll_container` would be a better name.

Reply:
Cool, Thanks for this, Updated it in the PR.